core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/provider.go

```diff
@@ -109,7 +109,7 @@ func NewLogProvider(lggr logger.Logger, poller logpoller.LogPoller, packer LogDa
 		threadCtrl:  utils.NewThreadControl(),
 		lggr:        lggr.Named("KeepersRegistry.LogEventProvider"),
 		packer:      packer,
-		buffer:      newLogEventBuffer(lggr, int(opts.LookbackBlocks), maxLogsPerBlock, maxLogsPerUpkeepInBlock),
+		buffer:      newLogEventBuffer(lggr, int(opts.LookbackBlocks), defaultFastExecLogsHigh),
```

Comment:
This is the first PR introducing this variable, in future PRs we will get this value in the log provider constructor and pass it over to log buffer constructor, i.e. this default value will be used only from within the buffer constructor in case the passed value is nil/0